app_dart/test/request_handlers/get_branches_test.dart

```diff
@@ -60,7 +60,7 @@ void main() {
     });
 
     test('returns branches matching regExps', () async {
-      githubBranches = <String>['flutter-1.1-candidate.1', 'master', 'test'];
+      config.flutterBranchesValue = 'flutter-1.1-candidate.1,master,test';
```

Comment:
Any reasons for not keeping the list of strings? I think branches are more readable as a list rather than a comma separated list.

Reply:
Let me try saving that info as an array.

Reply:
Oh, it supports only `String` type: https://github.com/keyonghan/cocoon/blob/master/app_dart/lib/src/datastore/cocoon_config.dart#L40

Reply:
Would it be possible to use memcache instead of datastore? it will be faster and we don't need to hit datastore for branching data.

Reply:
Actually here redisCache is used to cache this single value.  @CaseyHillers may have a better idea to choose redishCache over memcache.

Reply:
IIRC memcache was deprecated in the Dart AppEngine libraries when I implemented the redis caching (my initial approach was to implement memcache).

Reply:
Redis does only store strings, so `flutterBranches` getter would need a wrapper to convert the string to `List<String>`. There's some examples in the code base for how to serialize a value to bytes and serialize it back to cache it in redis.

Reply:
For example, https://github.com/flutter/cocoon/blob/master/app_dart/lib/src/datastore/cocoon_config.dart#L153 deserializes the service account which is a JSON object.